src/LuaPrinter.ts

```diff
@@ -163,8 +161,8 @@ export class LuaPrinter {
 
         return originalPos !== undefined && originalPos.line !== undefined && originalPos.column !== undefined
             ? new SourceNode(originalPos.line + 1, originalPos.column, this.sourceFile, chunks)
-            // tslint:disable-next-line:no-null-keyword
-            : new SourceNode(null, null, this.sourceFile, chunks);
+            : // tslint:disable-next-line:no-null-keyword
+              new SourceNode(null, null, this.sourceFile, chunks);
```

Comment:
This is kind of weird. Does the tslint:disable still work here?

Reply:
Looks like it does still work

Reply:
It indeed looks weird, but kind of makes sense, since comment is attached to `new SourceNode(...)` node. Instead we may use `tslint:disable-line`.
Issue: https://github.com/prettier/prettier/issues/2076.

Reply:
Personally, I'd prefer to stick with explicitly disabling specific rules instead of the whole line (the line could break a different rule still). The odd formatting isn't a huge deal and could be fixed in the future.

Reply:
It's possible to disable specific rule:
```ts
return originalPos !== undefined && originalPos.line !== undefined && originalPos.column !== undefined
    ? new SourceNode(originalPos.line + 1, originalPos.column, this.sourceFile, chunks)
    : new SourceNode(null, null, this.sourceFile, chunks); // tslint:disable-line:no-null-keyword
```

Reply:
Nice - didn't know that